express/extension.js

```diff
@@ -125,15 +125,18 @@ class Extension {
             let ac_nr_expired = ((session.access_token_validity - new Date().getTime()) / 1000) <= 120;
             if (ac_nr_expired) {
                 logger.debug(`Renewing access token for company ${companyId}`);
-                const renewTokenRes = await platformConfig.oauthClient.renewAccessToken();
+                const renewTokenRes = await platformConfig.oauthClient.renewAccessToken(session.access_mode === 'offline');
                 renewTokenRes.access_token_validity = platformConfig.oauthClient.token_expires_at;
                 session.updateToken(renewTokenRes);
                 await SessionStorage.saveSession(session);
                 logger.debug(`Access token renewed for company ${companyId}`);
             }
         }
-       
-        return new PlatformClient(platformConfig);
+        let platformClient = new PlatformClient(platformConfig);
+        platformClient.setExtraHeaders({
+            'x-ext-lib-version': version
```

Comment:
where is version format as we discussed? We cannot pass direct version number here.